### ISSUE TITLE
Feature request: SI prefixes notation

### ISSUE THREAD

**Issue by IvanSanchez** [antimatter-dimensions/notations#115]
I'd like to have notations using the [prefixes from the international system of units](https://en.wikipedia.org/wiki/Metric_prefix), i.e.:

> k M G T P E Z Y R Q B

Or their long forms:

> Kilo Mega Giga Tera Peta Exa Zetta Yotta Ronna Quecca Bundecca

Note that I'm including the [proposal](https://www.bipm.org/cc/CCU/Allowed/24/19-10_03_SI_prefixes_summary_RJC_Brown.pdf) for Ronna and Quecca, as well as the hypothetical Bundecca.

I guess that a nice approach would be to add this as a setting, and letting the `abbreviate` function use either the "standard" list (K M B T Qa Qt Sx Sp Oc No Dc, etc), the "short SI" list (k M G T P E Z Y R Q B) or the "long SI" list (Kilo Mega Giga Tera Peta Exa Zetta Yotta Ronna Quecca Bundecca). I don't see a need to add whole new notations, but rather change how the ones using the "standard" suffixes work.

There's also a [proposal to extend the SI prefixes up to 10^48](https://sci-hub.st/10.1016/j.measurement.2019.04.024), which boils down to 

> k M G T P E Z Y kY MY GY TY PY EY ZY YY

Arguably this could be extended like...

> k M G T P E Z Y R Q B kB MB GB TB PB EB ZB YB RB QB BB kBB MBB GBB TBB PBB EBB ZBB YBB RBB QBB BBB kBBB

...and later, add an exponent to the Bundeccas (to support those insane `1e9999999999` numbers), like

> MB² = MBB
> MB³ = MBBB
> MB⁴ = MBBBB

**Comment by dan-simon** [comment]
Yeah, this seems good. I think either SI list with an exponent to the Bundeccas would be a good thing for `abbreviate` to be able to do, so 1e9999999999 would be 1 B³⁰³⁰³⁰³⁰³ or 1 Bundecca³⁰³⁰³⁰³⁰³, and 1e10000000008 would be 1 GB³⁰³⁰³⁰³⁰³ or 1 Gigabundecca³⁰³⁰³⁰³⁰³.
